blog/2024/08/starting-workgroups.md

```diff
@@ -0,0 +1,64 @@
+
+It really helps to direct the efforts and accomplishing things faster if you have agenda/roadmap planned out. But a word of caution, don't plan too ahead in the future. It is best to decide small roadmaps lasting a month or so. This will also give enough chance to all people in your workgroup to sufficiently take part in more events than a tightly packed week long or sparse year long planned agenda.
+
+Ask the members if they prefer certain activities more than other activities. Take into account all opinions if possible, this will help in preparing/refreshing your agenda every month have new events, plan liked activities more and remove the ones with less/no response. You can post about agenda or upcoming events in other slack channels(of course respecting the decorum) and promote the events on the social handles to gather more eyes to your workgroup.
+
+Some positive things you could introduce is to appreciate new talent joining or appreciating new contributors each month. This helps in keeping the workgroup active, also keeps an positive vibe and the morale of both old and new members is raised.
+
+## Setting up a time
+
+We have members and you have planned a nice agenda with consultation of the members. The next step is to decide a time for the events such that you get the most participation and people also get a chance to digest things which they have attended. The best thing you can do is ask the people about the timings which they are most comfortable with. Add a poll for upcoming events with appropriate options for different times which takes in consideration of most timezones. If by any chance, the event gets cancelled or reschedule try to notify everyone about the update status as soon as possible to avoid any conflicts and save member's valuable time. Try revising the times quartely, half-yearly or yearly so that new and old members can adjust things as per their updated schedules.
+
+## What to do next with meetings
+
+You have now successfully organised an event and all members are happy with it too. There are some more points which can improve the already better things to the best.
+
+A meeting won't have at infinitium time for everyone, they will have a limited time window to discuss things. It is but obvious that not all topics will be covered, what one can do is at the end of the meeting one can ask that if someone wanted to talk about something but wasn't able to due to time restrictions or some topics are left from the original agenda and then you can add those topics for the next meetings. This way, topics will be covered and no one will be left unsatisfied.
+
+Additionally, you can also have a document in which you write down short but relevant minutes of meetings so if someone wants to take a look to get a better context for the past events they can and possibly leave some useful comments if they want.
+
+## General tips for being a workgroup leader
```

Comment:
I believe using the word "coordinator" instead of "leader" gives a better impression, as leader imposes a strong hierarchy that I am not sure fits with the theme of the informal and rather casual meetings of a workgroup.

Reply:
Nice, catch.